.travis.yml

```diff
@@ -6,6 +6,8 @@ rvm:
   - 2.4.6
   - 2.5.5
   - 2.6.3
+  - 2.6.5
```

Comment:
Can you replace version `2.6.3` with `2.6.5` instead? We try to keep only one build per minor version.

Reply:
@tegon thanks for feedback! 🤝 i will make this change

Reply:
@tegon done 👍 🎉